enterprise/wsproxy/wsproxysdk/wsproxysdk.go

```diff
@@ -508,7 +507,15 @@ func (c *Client) TailnetDialer() (*workspacesdk.WebsocketDialer, error) {
 		return nil, xerrors.Errorf("parse url: %w", err)
 	}
 	q := coordinateURL.Query()
-	q.Add("version", proto.CurrentVersion.String())
+	// The current version includes additions
+	//
+	// 2.1 GetAnnouncementBanners on the Agent API (version locked to Tailnet API)
+	// 2.2 PostTelemetry on the Tailnet API
+	// 2.3 RefreshResumeToken, WorkspaceUpdates
+	//
+	// Since resume tokens and telemetry are optional, and fail gracefully, and we don't use
+	// WorkspaceUpdates in the wsproxy, we ask for version 2.0 for maximum compatibility
+	q.Add("version", "2.0")
```

Comment:
Suggestion: We could move `"2.0"` to a const and the comment as well? This way it has a bit more exposure outside the package. Just thinking if it's a bit too hidden whilst making assumptions(?) about current business logic/requirements.